src/OpenLoco/src/Windows/DebugWindow.cpp

```diff
@@ -0,0 +1,112 @@
+#include "Graphics/Colour.h"
+#include "Graphics/Gfx.h"
+#include "Graphics/ImageIds.h"
+#include "Graphics/SoftwareDrawingEngine.h"
+#include "Graphics/TextRenderer.h"
+#include "Localisation/StringIds.h"
+#include "Objects/InterfaceSkinObject.h"
+#include "Objects/ObjectManager.h"
+#include "Ui/Widget.h"
+#include "Ui/Widgets/ButtonWidget.h"
+#include "Ui/Widgets/FrameWidget.h"
+#include "Ui/Widgets/ImageButtonWidget.h"
+#include "Ui/Widgets/LabelWidget.h"
+#include "Ui/Widgets/PanelWidget.h"
+#include "Ui/WindowManager.h"
+#include <vector>
+
+namespace OpenLoco::Ui::Windows::Debug
```

Comment:
Are we wanting to keep this in or was it just for testing purposes?

Reply:
I think we should keep that for as long those refactors are going on, it would help if I had always a window I can hack around, the one unfortunate thing is that its opened by a shortcut.